### DISCUSSION TITLE
Migrate form WF to CoreWF

### DISCUSSION THREAD

**Discussion by rnagr** [UiPath/corewf#204] [Q&A]
Can you please guide me to migrate custom workflow activity which is created by inheriting SequencialWorkflowActivity of WF3 to CoreWF?

**Comment by dmetzgar** [comment]
There's a bunch of documentation on how to migrate from WF3 to WF4 here: https://docs.microsoft.com/en-us/dotnet/framework/windows-workflow-foundation/migration-guidance

The only thing not available is the interop activity.
